zivid_camera/srv/Capture2D.srv

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Just curious: Why are we not using `std_srvs::Empty` here? Is it because we want the service to be named as what it does? Or is it in case the service will be expanded in the future?

I think it is nice that it is named as what it does, but I guess also there is a trade-off where we want to use as many standard ROS message/service types as possible.

Reply:
It is mainly for future proofing. For example, it could be that we wanted to add a return value which can be used to identify which point cloud belongs to which capture.

Reply:
Yes, I think that is a good reason (in addition to naming the intent of the message). 

The service return value could in theory also be the color image itself. Don’t think we should change that of course, since that is not how we do it for the 3D capture.